languages/nl-NL.js

```diff
@@ -165,6 +165,7 @@ module.exports = class extends Language {
       CLICK_HERE: 'Klik hier',
 
       // Fun Category
+      THEMSELVES: 'themselves',
```

Comment:
Is that the correct translation? Or just default english?

Reply:
Dutch translator has no time atm. so i just put the english translation in there to prevent error messages